src/clusterfuzz/_internal/datastore/data_handler.py

```diff
@@ -385,11 +385,21 @@ def format_issue_information(testcase, format_string):
   sanitizer_options_string = ' '.join(sanitizer_options)
   bazel_test_args = _get_bazel_test_args(arguments, sanitizer_options)
 
+  # Multi-target binaries.
```

Comment:
OK, I'm not sure I grok GFT well enough to understand this part. Could you explain? Are GFT fuzz tests fuzzable with LF and GFT?

Reply:
This is for reproduction instructions. A GFT fuzzer is of the form `binary@test_name`. This is the name of the wrapper shell script which ClusterFuzz is executing under the hood, but for user reproduction steps, we want to provide template variables to parse these out. 

GFT tests are not fuzzable with LF today.